src/core/file/fileManipulate.ts

```
@@ -88,39 +93,25 @@ class PythonManipulator extends BaseManipulator {
     };
+
     let result = '';
     const pairs: [number, number][] = [];
     let prevQuote = 0;
     while (prevQuote < content.length) {
-      const openingQuote: number = content.slice(prevQuote + 1).search(/(?<!\\)(?:"|'|'''|""")/g) + prevQuote + 1;
+      const openingQuote = content.slice(prevQuote + 1).search(/(?<!\\)(?:"|'|'''|""")/g) + prevQuote + 1;
       if (openingQuote === prevQuote) break;
 
-      let closingQuote: number;
+      let closingQuote;
```

Biome

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)